src/test/journalTest.sh

```diff
@@ -359,3 +359,11 @@ test_packageLoggingGuess() {
   assertTrue "rlJournalStart survives garbage in TEST" "rlJournalStart"
   assertFalse "No <pkgdetails> tag when TEST is garbage" "rlJournalPrint | grep -q '<pkgdetails>'"
 }
+
+test_phaseNameIn_rlReport(){
+    export BEAKERLIB_COMMAND_REPORT_RESULT=echo
+    journalReset
+
+    silentIfNotDebug "rlPhaseStartTest '//some/-phase//na--me-'"
+    assertTrue "Phase name is correctly converted when passing value to rlReport in rlPhaseEnd" "rlPhaseEnd | grep '^some-phase-na-me- '"
```

Comment:
Looking at the example, wouldn't it be better to strip the dashes trailing at the end as well?

Reply:
I don't think it's necessary from the perspective of the `tmt-report-result` script (or other `BEAKERLIB_COMMAND_REPORT_RESULT` scripts). The trailing dashes shouldn't affect getopts parsing in any way. However, I agree that removing them does make the output look neater.

I've added another sed which removes them.